README.md

```diff
@@ -73,6 +73,7 @@ After you're finished please remove all the comments and instructions!
 - Vet clinic database
 - animals table
 - Querying data
+- Species column
```

Comment:
- Please note that the MIT Lisence link provided is invalid. 😔 Kindly provide a valid license link 👍🏼